pkg/operator/status.go

```diff
@@ -306,6 +307,11 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		if isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolUpdating) {
 			updating = true
 		}
+		if err != nil {
+			klog.Errorf("error on Upgrading Progression %w", err)
+			return err
```

Comment:
suggestion (non-blocking): The `error on Upgrading Progression` context could be useful to the caller. Do this instead:

```golang
progErr := fmt.Errorf("error on Upgrading Progression %w", err)
klog.Error(progErr)
return progErr
```